doc/onboarding/view/step2.md

```diff
@@ -0,0 +1,82 @@
+
+
+## Step 2/5 : Share anonymously data
+
+### URI
+
+`:userID/share`
```

Comment:
@misstick Unless I am mistaken, the commit https://github.com/cozy/cozy-proxy/pull/300/commits/937ff9041a3b97154b8c3e8191a97d290c81c240 that you just gave doesn't solve the naming issue here.